src/test/java/org/fundacionjala/sfdc/stepdefinition/CommonAssertions.java

```diff
@@ -11,14 +13,16 @@
  */
 public class CommonAssertions {
     private SFCommonObjects commonPage;
+    private Helper helperToHook;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "helperToHook" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)